packages/grafana-ui/src/components/DateTimePickers/TimeRangeInput.tsx

```diff
@@ -77,13 +79,30 @@ export const TimeRangeInput = ({
     onChange({ from, to, raw: { from, to } });
   };
 
+  const overlayRef = createRef<HTMLElement>();
```

Comment:
Why do we use `createRef` for the overlayRef but `useRef` for the button ref? 🤔

Reply:
I don't know, I must have changed it accidentally when re-implementing from the other component 🤷 I'll change it to be consistent with `TimeRangePicker`, thanks for spotting it